cmd/simulator/load/loader.go

```diff
@@ -73,10 +82,11 @@ func CreateLoader(ctx context.Context, config config.Config) (*WorkerGroup, erro
 	client := clients[0]
 	chainID, err := client.ChainID(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("failed to fetch chainID: %w", err)
+		return fmt.Errorf("failed to fetch chainID: %w", err)
 	}
 	signer := types.LatestSignerForChainID(chainID)
 
+	log.Info("Creating transaction sequences...")
 	txGenerator := func(key *ecdsa.PrivateKey, nonce uint64) (*types.Transaction, error) {
```

Comment:
It seems this txGenerator func can just be a series of options/params for `GenerateTxSequences`